src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $cells-gap;
+
+  padding: 82px $calendar-padding;
+  width: calc(7 * $cell-width + 6 * $cells-gap + 2 * $calendar-padding);
+
+  &__day {
+    width: $cell-width;
+    height: $cell-height;
+    text-align: center;
+    background-color: #eee;
+
+    border: 1px solid #000;
+
+    transition: transform 0.5s ease;
+
+    &:hover {
+      cursor: pointer;
+
+      // background-color: #ffbfcb; in the test background is grey =(
```

Comment:
In the future - it is a bad practice to keep the commented-out code. If you need it - try transforming it into some meaningful comment. If it is not needed - just remove it.